configuration/external_parameters.rst

```diff
@@ -13,93 +13,25 @@ you to easily do this.
 Environment Variables
 ---------------------
 
-Symfony will grab any environment variable prefixed with ``SYMFONY__`` and
-set it as a parameter in the service container. Some transformations are
-applied to the resulting parameter name:
-
-* ``SYMFONY__`` prefix is removed;
-* Parameter name is lowercased;
-* Double underscores are replaced with a period, as a period is not
-  a valid character in an environment variable name.
-
-For example, if you're using Apache, environment variables can be set using the
-`SetEnv`_ directive with the following ``VirtualHost`` configuration:
-
-.. code-block:: apache
-
-    <VirtualHost *:80>
-        ServerName      Symfony
-        DocumentRoot    "/path/to/symfony_2_app/web"
-        DirectoryIndex  index.php index.html
-        SetEnv          SYMFONY__DATABASE__USER user
-        SetEnv          SYMFONY__DATABASE__PASSWORD secret
-
-        <Directory "/path/to/symfony_2_app/web">
-            AllowOverride All
-            Allow from All
-        </Directory>
-    </VirtualHost>
-
-For Nginx web servers, the environment variables can be set with the `fastcgi_param`_
-directive. For example, in the configuration file where the ``fastcgi_params``
-file is included:
-
-.. code-block:: nginx
-
-    server {
-        server_name domain.tld www.domain.tld;
-        root /var/www/project/web;
-
-        location / {
-            try_files $uri /app.php$is_args$args;
-        }
-
-        location ~ ^/app\.php(/|$) {
-            fastcgi_pass unix:/var/run/php5-fpm.sock;
-            fastcgi_split_path_info ^(.+\.php)(/.*)$;
-            include fastcgi_params;
-            fastcgi_param SCRIPT_FILENAME $realpath_root$fastcgi_script_name;
-            fastcgi_param DOCUMENT_ROOT $realpath_root;
-            fastcgi_param SYMFONY__DATABASE__USER user;
-            fastcgi_param SYMFONY__DATABASE__PASSWORD secret;
-            internal;
-        }
-
-        # ...
-    }
+.. versionadded:: 3.2
+    ``env()`` parameters were introduced in Symfony 3.2.
 
-.. note::
-
-    The examples above are for an Apache and Nginx configuration. However, this
-    will work for any web server which supports the setting of environment
-    variables.
-
-    Also, in order for your console to work (which does not use a web server),
-    you must export these as shell variables. On a Unix system, you can run
-    the following:
-
-    .. code-block:: terminal
+You can reference environment variables by using special parameters named after
+the variables you want to use enclosed between ``env()``. Their actual values
+will be resolved at runtime, so that dumped containers can be reconfigured
```

Comment:
Per request they are resolved once. But if you change the value, during the next request you will see the new value (again resolved only once during this request). Should we make that more clear somehow?